tests/e2e/support/mod.rs

```diff
@@ -9,9 +9,22 @@ mod mock_client;
 pub mod normalize;
 pub mod scarb;
 
+use serde_json::Value;
+
 pub use self::cursor::cursors;
 pub use self::mock_client::MockClient;
 
+/// Merges `b`'s kv pairs into `a` (non recursively), potentially overriding the previous values
+pub(crate) fn merge_json_flat(a: &mut Value, b: Value) {
```

Comment:
nit: please move this to the end of this file. also worth adding `#[doc(hidden)]`